profile/README.md

```diff
@@ -0,0 +1,18 @@
+
+## U.S. IOOS&reg; - Eyes on the Ocean, Coasts, and Great Lakes&trade;
+
+**Our mission:** To produce, integrate, and communicate high quality ocean, coastal and Great Lakes information that meets the safety, economic, and stewardship needs of the Nation.
+
+The IOOS Data Management and Cyberinfrasctructure (DMAC) program maintains a collection of open source software packages, documentation, and utilities to facilitate data distribution and cyberinfrastructure implementation within our community of practice.  These resources are all contained within the IOOS GitHub organization.
+
+For more information about U.S. IOOS visit: [https://ioos.noaa.gov](https://ioos.noaa.gov).
+
+### How To Request Changes to IOOS' GitHub Resources:
+
+A short list of common community requests.  
+
+- **Add new contributors to IOOS repos** - email micah dot wengren at noaa dot gov, melissa dot zweng at noaa dot gov, or mathew dot biddle at noaa dot gov.  Please include the following:
```

Comment:
For adding new contributors to IOOS repos - could we make another template for that? I'm hoping to avoid "send an email to these people".